pkg/datastore/test/counters.go

```diff
@@ -174,7 +174,7 @@ func UpdateRelationshipCounterTest(t *testing.T, tester DatastoreTester) {
 	require.Len(t, filters, 1)
 	require.Equal(t, "somedocfilter", filters[0].Name)
 	require.Equal(t, 1234, filters[0].Count)
-	require.Equal(t, updatedRev, filters[0].ComputedAtRevision)
+	require.True(t, updatedRev.Equal(filters[0].ComputedAtRevision))
```

Comment:
Add a comment here as to why Equal is not used

Reply:
https://github.com/authzed/spicedb/pull/1951/commits/dc450332fa3d03ed97d176a50d1e33686e9fdf13